hello-world/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "hello-world"
+version = "0.0.1"
+dependencies = [
+    "build == 0.10.0"
```

Comment:
`build` should probably be in a separate optional dependency as it isn't needed at run time.